src/chunk_append/exec.c

```diff
@@ -723,6 +723,10 @@ constify_param_mutator(Node *node, void *context)
 	if (node == NULL)
 		return NULL;
 
+	/* Don't descend into subplans to constify their parameters */
```

Comment:
```suggestion
	/* Don't descend into subplans to constify their parameters, because they may nit be valid yet */
```

and any info on why would also be helpful